.github/workflows/graalpy-notebook-example.yml

```diff
@@ -20,21 +20,19 @@ jobs:
     timeout-minutes: 40
     strategy:
       matrix:
-        include:
-          - version: '22.3.1' # temporarily locked to 22.3.1
-            java-version: '17'
+        java-version: ['21']
     steps:
       - uses: actions/checkout@v3
       - uses: graalvm/setup-graalvm@v1
         with:
-          version: ${{ matrix.version }}
           java-version: ${{ matrix.java-version }}
+          distribution: 'graalvm'
           components: 'python'
```

Comment:
You can't install 'python' component on GraalVM for JDK 21. We should keep this demo testing on java-version: '17'.

Reply:
That's one option. As I understand, you need to install GraalPy JVM standalone 23.1.1 and export GRAALPY_HOME to to using GitHub actions workflow.

Reply:
If we decide to update this demo to a newer version of GraalPy, we no longer can/should use the `setup-graalvm` action, but you may be able to use `setup-python`, which [supports GraalPy](https://github.com/actions/setup-python/pull/694).